extra/help/lint/coverage/coverage.factor

```diff
@@ -78,30 +109,14 @@ M: word-help-coverage summary
     [ should-define ] [ word-defines-sections ] bi diff ;
 PRIVATE>
 
-GENERIC: print-coverage ( coverage-seq -- )
-M: sequence print-coverage
-    [
-        [ print-coverage ] each
-    ] [
-        [ [ 100%-coverage?>> ] count ] [ length ] bi /f
-        100 *
-        "\n%3.1f%% of words have complete documentation\n"
-        printf
-    ] bi ;
-
-M: word-help-coverage print-coverage
-    summary print ;
-
 GENERIC: <word-help-coverage> ( word -- coverage )
 M: word <word-help-coverage>
-    dup
-    [ missing-sections ]
-    [ empty-examples? ] bi
+    dup [ missing-sections ] [ empty-examples? ] bi
     2dup 2array { { } f } =
     word-help-coverage boa ; inline
 
 M: string <word-help-coverage>
-    loaded-vocab-names resolve-name-in <word-help-coverage> ; inline
+    find-word <word-help-coverage> ; inline
```

Comment:
Don't you think this doesn't work well for ``"search" find-word``?

My version did a restart when there were several potential options rather than returning only the first.  Also, we probably want it to work for qualified strings like ``"sequences:nth" find-word``.